src/usr/local/buildpack/tools/python.sh

```diff
@@ -22,6 +22,7 @@ if [[ -z "${tool_path}" ]]; then
   file=/tmp/python.tar.xz
 
   ARCH=$(uname -p)
+  # shellcheck source=/dev/null
   CODENAME=$(. /etc/os-release && echo "${VERSION_CODENAME}")
```

Comment:
We should extract this to a util function

Reply:
I can do that, we have the utils file included everywhere.